src/egon/data/datasets/gas_grid.py

```diff
@@ -293,101 +293,120 @@ def insert_gas_buses_abroad(scn_name="eGon2035"):
     gas_carrier = "CH4"
     # Connect to local database
     engine = db.engine()
-    
-    db.execute_sql(
-        f"""
-    DELETE FROM grid.egon_etrago_bus WHERE "carrier" = '{gas_carrier}' AND
-    scn_name = '{scn_name}' AND country != 'DE';
-    """
-    )
 
-    # Select the foreign buses
-    gdf_abroad_buses = central_buses_pypsaeur(sources, scenario=scn_name)
-    gdf_abroad_buses = gdf_abroad_buses.drop_duplicates(subset=["country"])
+    # for the eGon100RE scenario the CH4 buses are created by electrical_neighbours_egon100()
+    # therefore instead of created the buses, for this scenario the buses are just read.
+    if scn_name == "eGon100RE":
+        gdf_abroad_buses = geopandas.read_postgis(
+            f"""
+            SELECT * FROM grid.egon_etrago_bus WHERE "carrier" = '{gas_carrier}' AND
+            scn_name = '{scn_name}' AND country != 'DE';
+            """,
+            con=engine,
+            crs=4326,
+        )
+        gdf_abroad_buses.drop_duplicates(
+            subset="country", keep="first", inplace=True
+        )

```

Comment:
Do you know why there are duplicates in there?

Reply:
There are two CH4 buses for GB and DK coming from the pypsa-eur dataset. We have the same case for electricity in these two countries.